src/common/remote/grpc/layers/auth.rs

```diff
@@ -31,31 +31,44 @@ impl AuthLayer {
         auth_params: HashMap<String, String>,
         id: String,
     ) -> Self {
-        AuthLayer::login_task(auth_plugin.clone(), server_list, auth_params, id);
+        Self::init(auth_plugin.clone(), server_list, auth_params, id);
         Self { auth_plugin }
     }
 
-    fn login_task(
+    fn init(
         auth_plugin: Arc<dyn AuthPlugin>,
         server_list: Vec<String>,
         auth_params: HashMap<String, String>,
         id: String,
     ) {
-        let _span_enter = debug_span!("auth_task", id = id).entered();
-        let auth_context = AuthContext::default().add_params(auth_params);
-        // auth_plugin.login(server_list.clone(), auth_context.clone());
+        let (tx, rx) = oneshot::channel::<()>();
```

Comment:
what about `let (tx, rx) = std::sync::mpsc::channel();` not need the `thread::spawn(move || rx.blocking_recv());`